### PR TITLE
API-1297 JUnit integration tests

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vajira.max.gov/browse/API-264

### CODE REVIEW

File: community-care-eligibility-tests/src/test/java/gov/va/api/health/communitycareeligibility/tests/SearchValidationIT.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        String.format(
+            "v0/eligibility/search?patient=%s&serviceType=%s&extendedDriveMin=%s",
+            systemDefinition().patient(), "PrimaryCare", -1);
+    search(request, 400).expectValid(ErrorResponse.BadRequest.class);
+  }
+
+  @Test
+  void missingPatient() {
+    String request = String.format("v0/eligibility/search?serviceType=%s", "PrimaryCare");
+    search(request, 500).expectValid(ErrorResponse.InternalServerError.class);
```
Comment (jblefkove-va) [inline community-care-eligibility-tests/src/test/java/gov/va/api/health/communitycareeligibility/tests/SearchValidationIT.java:41]: This is lame... I will make a follow-up story for a better status code.

File: community-care-eligibility-tests/src/main/java/gov/va/api/health/communitycareeligibility/tests/SystemDefinitions.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        .contentType("application/json")
+        .build();
+  }
+
+  private static SystemDefinition lab() {
+    return SystemDefinition.builder()
+        .cce(
+            serviceDefinition(
+                "community-care-eligibility",
+                "http://blue.lab.lighthouse.va.gov",
```
Comment (jblefkove-va) [inline community-care-eligibility-tests/src/main/java/gov/va/api/health/communitycareeligibility/tests/SystemDefinitions.java:32]: I copied this from Facilities, but... shouldn't this be `https`?

Comment (bschofield-va) [reply community-care-eligibility-tests/src/main/java/gov/va/api/health/communitycareeligibility/tests/SystemDefinitions.java:32]: yes, there isn't even an HTTP listener on the ALBs.

File: community-care-eligibility-tests/src/test/java/gov/va/api/health/communitycareeligibility/tests/SearchIT.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+import static gov.va.api.health.communitycareeligibility.tests.Requestor.search;
+import static gov.va.api.health.communitycareeligibility.tests.SystemDefinitions.systemDefinition;
+
+import gov.va.api.health.communitycareeligibility.api.CommunityCareEligibilityResponse;
+import org.junit.jupiter.api.Test;
+
+public class SearchIT {
+  @Test
+  void audiology() {
```
Comment (bschofield-va) [inline community-care-eligibility-tests/src/test/java/gov/va/api/health/communitycareeligibility/tests/SearchIT.java:11]: Any value in checking to see if you have more than an empty results?

Comment (jblefkove-va) [reply community-care-eligibility-tests/src/test/java/gov/va/api/health/communitycareeligibility/tests/SearchIT.java:11]: Yes, but I need to think on it. The different test patients complicate things. Some have a static E&E code that makes them automatically eligible for every service (and they won't have any nearby facilities), but for the ones that don't, eligibility will depend on what facilities are nearby and what services they offer.

**Pull request merged**